vsintegration/tests/FSharp.Editor.Tests/CodeFixes/ChangeEqualsInFieldTypeToColonTests.fs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+module FSharp.Editor.Tests.CodeFixes.ChangeEqualsInFieldTypeToColonTests
```

Comment:
TBH I am not sure if the visitor will work for anonymous records out of the box or not.
Better add a test case for anonymous records too, and adjust impl if needed.

`let f (x:{|A=string|}) = x`
should be codefixed to
`let f (x:{|A:string|}) = x`

Reply:
Yep, it's a different thing apparently, VisitRecordDefn x VisitTypeAbbrev. 
Added.